src/components/folder/LinkCards.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import addIcon from '../../images/kebab.svg';
+import starIcon from '../../images/star.svg';
+import noImgIcon from '../../images/noCardImg.png';
+import '../../style/folderContent.css';
+
+const LinkCard = ({ link }) => {
+  return (
```

Comment:
```suggestion
const LinkCard = ({ link }) => {
  const {
    url,
    image_source,
    title,
    timePassed,
    description,
    formattedDate,
  } = link;
  return (
```

이런식으로 디스트럭처링 할당을 해보시면 어떨까요?
물론 값을 명시적으로 표현하려는 목적이라면 link.url 등 프로퍼티 접근자로 꺼내와도 됩니다!